src/bridge/ExposureNotification/ExposureNotificationAdapter.android.ts

```diff
@@ -45,16 +45,22 @@ export default function ExposureNotificationAdapter(exposureNotificationAPI: Exp
       return [];
     },
     getExposureWindowsAndroid: async (diagnosisKeysURLs: string[]) => {
-      captureMessage('getExposureWindows');
+      log.debug({category: 'exposure-check', message: 'getExposureWindows'});
```

Comment:
Any need for this?

Reply:
Probably not - it was something I added when the function call below was crashing. I will remove it.